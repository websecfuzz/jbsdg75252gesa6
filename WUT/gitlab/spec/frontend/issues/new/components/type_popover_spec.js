import { shallowMount } from '@vue/test-utils';
import TypePopover from '~/issues/new/components/type_popover.vue';

describe('Issue type info popover', () => {
  let wrapper;

  function createComponent() {
    wrapper = shallowMount(TypePopover);
  }

  it('renders', () => {
    createComponent();

    expect(wrapper.element).toMatchSnapshot();
  });
});
