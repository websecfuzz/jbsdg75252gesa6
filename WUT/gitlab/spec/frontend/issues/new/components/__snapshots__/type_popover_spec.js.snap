// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Issue type info popover renders 1`] = `
<span
  class="gl-ml-2"
  id="reference-0"
>
  <help-icon-stub
    id="reference-1"
  />
  <gl-popover-stub
    container="popovercontainer"
    cssclasses=""
    target="issue-type-info"
    title="Issue types"
    triggers="focus hover"
  >
    <ul
      class="gl-list-none gl-m-0 gl-p-0"
    >
      <li
        class="gl-mb-3"
      >
        <div
          class="gl-font-bold"
        >
          Issue
        </div>
        <span>
          For general work
        </span>
      </li>
      <li>
        <div
          class="gl-font-bold"
        >
          Incident
        </div>
        <span>
          For investigating IT service disruptions or outages
        </span>
      </li>
    </ul>
  </gl-popover-stub>
</span>
`;
